zipkin-server/src/main/resources/zipkin-server.yml

```diff
@@ -83,13 +83,18 @@ zipkin:
     # - .*example2.com
     # Default is "match all websites"
     instrumented: .*
+  server:
```

Comment:
I suppose of it's only the query API that this applies to (which is the case at the moment) then this would make sense (reusing `zipkin.query`.